M/MUMPS/MUMPS_seq@5/build_tarballs.jl

```diff
@@ -15,7 +15,7 @@ using BinaryBuilder, Pkg
 
 name = "MUMPS_seq"
 upstream_version = v"5.6.2"
-version_offset = v"0.0.1" # reset to 0.0.0 once the upstream version changes
+version_offset = v"0.0.2" # reset to 0.0.0 once the upstream version changes
```

Comment:
Won't Yggdrasil just publish new versions as +1, +2, etc? Unless we are patching the package, I thought we wouldn't need to change the version_offset.

Reply:
Yes, but for Coin packages and its dependencies, we multiply all version numbers by 100:
https://github.com/JuliaPackaging/Yggdrasil/blob/master/M/MUMPS/MUMPS_seq%405/build_tarballs.jl#L19-L21
We often recompile a package (like Ipopt) with updated dependencies (MUMPS, SPRAL, ...).
Without this trick, if the new version of the dependency breaks the package, all the Coin ecosystem is broken... (cc @odow :stuck_out_tongue:).